scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -729,6 +759,104 @@ private void removeJobRuntimeData(Session session, long jobId) {
         session.getNamedQuery("deleteSelectorData").setParameter("jobId", jobId).executeUpdate();
     }
 
+    private void deleteInconsistentData(Session session) {
```

Comment:
With the previous implementation of the housekeeping we had cases where tasks existed without any job parent. Since the new mechanism handles that differently, @fviale maybe I can remove it ?